cfgov/v1/models/base.py

```diff
@@ -45,8 +65,57 @@ class CFGOVPage(Page):
         MultiFieldPanel(Page.promote_panels, "Page configuration"),
         FieldPanel('tags', 'Tags'),
         FieldPanel('authors', 'Authors'),
+        MultiFieldPanel([
+            FieldPanel('view_more_label', 'View More Label'),
+            FieldPanel('view_more_url', 'View More URL'),
+            FieldPanel('related_limit', 'Limit'),
+            FieldRowPanel([
+                FieldPanel('is_relating_posts', classname='col4'),
+                FieldPanel('is_relating_newsroom', classname='col4'),
+                FieldPanel('is_relating_events', classname='col4')
+            ])
+        ], heading='Related Posts')
     ]
 
+    # After all search types are migrated to Wagtail this should relate pages
+    # based on tags.
+    @property
+    def related_posts(self):
+        # After all search types are migrated to Wagtail, comment out below. If
+        # we decide we'd like to use the more_like_this feature of
+        # Elasticsearch, we can always revert back to this.
+        search_types = []
+        for search_type in ['posts', 'newsroom', 'events']:
+            if eval('self.is_relating_%s' % search_type):
+                search_types.append(search_type)
+
+        results = []
+        try:
+            # Gets an ES document across all types by the slug of the page.
+            document = get_document('_all', self.slug)
+            for search_type in search_types:
+                results += more_like_this(document, search_types=search_type,
```

Comment:
I'm just wondering if we need an if statement to check whether `None` or null is returned for the particular type.